HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -315,14 +319,59 @@ wait_server(){
 	echo "server is ready!"
 }
 
+edit_model(){
+    cp -r $1 $TMPDIR/local_model_repo/
+    IFS='/' read -ra ADDR <<< "$1"
+    CONFIG=$TMPDIR/local_model_repo/${ADDR[-1]}/config.pbtxt
+
+    echo "" >> "$CONFIG"
```

Comment:
this would be a little easier to read (and a little more efficient) if cat (heredoc) were used, e.g. https://github.com/cms-sw/cmssw/blob/3d1300215171ea99be65905a09849751fcdbd5ed/FWCore/Integration/test/test_finalpath.sh#L19-L27
(the downside is that the text body can't be indented to match the rest of the function, but I think it's a worthwhile tradeoff)

Reply:
I switched to this format